docs/source/index.md

```diff
@@ -4,3 +4,60 @@ Standard Architecture for Trusted Research Environments (TREs)
 ```{toctree}
 contributors.md
 ```
+
+## Introduction
+
+### Trusted Research Environments
+
+> What are they, how many exist, the broad categories
+
+### About the authors
+
+> Our respective development and use of TREs, with brief descriptions of the three examples
+
+### A Standard Architecture for TREs
+
+> Motivation: Why a TRE standard is needed/ useful and a description of the broader SATRE project, conception and goals
+
+### Standard aims
+
+> What this document intends to do (and what it doesn't), the level of detail we aim for contrasted with other technical standards
+
+## Required TRE features
+
+> Crowdsourced list of functionality from TRE developers survey that were considered the most important/ <u>essential components in order to define a software as a TRE</u>. Each of these should be described in prose and where appropriate examples of these features present in existing TRE implementations provided. Where features vary across existing TREs this should be highlighted.
+
+### Feature A
+### Feature B
+## Optional TRE features
+
+> Also croudsourced from the survey, but this time all the features that were considered less important and/or are not needed in order to simply define something as a TRE (for example this might include something like HPC-connectability).
+
+### Feature X
+### Feature Y
+
+## Roles
+
+> A TRE conforming to the SATRE standard should provide a broadly similar experience for stakeholders operating in each of these defined roles.
+
+### TRE users
+
+> The researchers working on projects that involve logging into a TRE to access data. The document will explain that user experience of the platform and associated documentation should feel similar across TREs conforming to SATRE standard.
+
+### TRE admins
```

Comment:
At some point I think we'll need to rename this since from the description this refers to administration of the technical/compute infrastructure, and not administering the TRE itself (e.g. managing users and projects).

Reply:
I think this comment has some important points that I've added